src/databases/FieldViewXDB/CMakeLists.txt

```diff
@@ -5,6 +5,19 @@
 # All other uses are strictly forbidden.
 #*****************************************************************************
 
+# Check for plugin extension
```

Comment:
I am not entirely sure what this block needed to be added for.

Is it telling VisIt to look for files with a given extension for this plugin?

Reply:
It is only used by FieldViewXDB, so that is why I moved it out of the root CMakelists.txt file.
I didn't dig further than that. The goal of the reorg step is to simplify root CMakelists.txt.